src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -835,7 +836,7 @@ protected function getContainerBuilder()
      * @param string           $class     The name of the class to generate
      * @param string           $baseClass The name of the container's base class
      */
-    protected function dumpContainer(ConfigCache $cache, ContainerBuilder $container, $class, $baseClass)
+    protected function dumpContainer(ConfigCache $cache, ContainerBuilder $container, $class, $baseClass, bool $asFiles = true)
```

Comment:
is this ever passed as `false` ?

Reply:
if you change the $asFile that are hardcoded... yes :) I just fixed the other side of an hardcoded path :joy:

Reply:
Instead of an argument, this should be a parameter to me.
That would fix the BC break and make this easier to configure I believe.

Reply:
a DIC parameter you mean?

Reply:
yes

Reply:
Fixed